static/js/pages/instances/InstanceDetailPanel.tsx

```diff
@@ -30,107 +50,183 @@ const InstanceDetailPanel: FC<Props> = ({ instance, notify, onClose }) => {
+                  isStopping={stopping.includes(instance.name)}
                 />
               </td>
             </tr>
             <tr>
-              <th>Type</th>
-              <td>{instance.type}</td>
+              <th className="u-text--muted">Description</th>
+              <td>{instance.description ? instance.description : "-"}</td>
             </tr>
             <tr>
-              <th>Architecture</th>
-              <td>{instance.architecture}</td>
+              <th className="u-text--muted">Type</th>
+              <td>
+                {
+                  instanceCreationTypes.filter(
+                    (item) => item.value === instance.type
+                  )[0].label
+                }
+              </td>
             </tr>
             <tr>
-              <th>Image</th>
+              <th className="u-text--muted">IPv4</th>
               <td>
-                <Row>
-                  <Col
-                    size={4}
-                    className="u-truncate"
-                    title={instance.config["image.description"]}
-                  >
-                    {instance.config["image.description"]}
-                  </Col>
-                </Row>
+                {ip4Addresses && ip4Addresses.length > 0 ? (
+                  <>
+                    {ip4Addresses.slice(0, ip4DisplayCount)}
+                    {ip4DisplayCount < ip4Addresses.length && (
+                      <Button
+                        appearance="link"
+                        className="u-no-margin--bottom"
+                        small
+                        onClick={() => setIp4DisplayCount(ip4Addresses.length)}
+                      >
+                        Show all
+                      </Button>
+                    )}
+                  </>
+                ) : (
+                  "-"
+                )}
               </td>
             </tr>
             <tr>
-              <th>Date created</th>
+              <th className="u-text--muted">IPv6</th>
+              <td>
+                {ip6Addresses && ip6Addresses.length > 0 ? (
+                  <>
+                    {ip6Addresses.slice(0, ip6DisplayCount)}
+                    {ip6DisplayCount < ip6Addresses.length && (
+                      <Button
+                        appearance="link"
+                        className="u-no-margin--bottom"
+                        small
+                        onClick={() => setIp6DisplayCount(ip6Addresses.length)}
+                      >
+                        Show all
+                      </Button>
+                    )}
+                  </>
+                ) : (
+                  "-"
+                )}
+              </td>
+            </tr>
+            <tr>
+              <th className="u-text--muted">Architecture</th>
+              <td>{instance.architecture}</td>
+            </tr>
+            <tr>
+              <th className="u-text--muted">PID</th>
```

Comment:
Did you find out why the PID is important to display it here?

Reply:
No, as I said, I assume it's there for a reason, so I don't really want to question that. I might quickly ask it out of curiosity during the next UX/UI sync, and that's all.